WORKSPACE

```diff
@@ -436,10 +436,6 @@ haskell_register_ghc_nixpkgs(
         "-fexternal-dynamic-refs",
     ] + (["-g3"] if enable_ghc_dwarf else ([
         "-optl-unexported_symbols_list=*",
-        "-optc-mmacosx-version-min=10.14",
```

Comment:
this caused some errors, need to take another look. That said, the warnings also never appeared to cause any issues but if possible, I’d still like to get rid of them.